article/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path('',views.artciles,name='articles'),
+    path('create/',views.createArtcile,name='create-article'),
```

Comment:
name="create-article" -> create minus article
Soln-> create_article

Reply:
as did in the feed.urls with minus . we have to maintain consistency so we have to keep it that way 
see - feed.urls.py
```
path('', views.mumbles, name="mumbles"),
     path('create/', views.createMumble, name="mumble-create"),
     path('remumble/', views.remumble, name="mumble-remumble"),
     path('vote/', views.updateVote, name="posts-vote"),

     path('<str:pk>/comments/', views.mumbleComments, name="mumble-comments"),
     
```

Reply:
see - feed/views.py 

```
@api_view(['POST'])
def createMumble(request):
    user = request.user
    data = request.data
```